### PR TITLE
🚀 Release ➾ v0.28.0

### PR DESCRIPTION
**Pull request by mweichert**
# 🌮 Taqueria PR

## 🪁 Description

Release v0.28.0

## 🪂 Pre-Merge Checklist (Definition of Done)

🚦 Required to merge:

- [ ] ⛱️ I have completed this PR template in full and updated the title appropriately
- [ ] ⛵ My code builds cleanly, and I have manually tested the changes
- [ ] 🏄‍♂️ Another team member has built this branch and done manual testing on the change
- [ ] 🏖️ New and existing unit tests pass locally and in CI
- [ ] 🔱 The test plan has been implemented and verified by an SDET
- [ ] 🦀 Automated tests have been written and added to this PR
- [ ] 🐬 I have commented my code, particularly in hard-to-understand areas
- [ ] 🤿 Corresponding changes have been made to all documentation
- [ ] 🐚 Required changes to the VScE have been made
- [ ] 🪸 Required updates to scaffolds have been made
- [ ] 🚢 The release checklist has been completed

### CODE REVIEW

**Review by michaelkernaghan — COMMENTED** [review]
looks like the same PR as for the pre-release but with the version number changed to be a production release number

**Review by egarson — APPROVED** [review]

**Review by GImbrailo — APPROVED** [review]
LGTM

**Review by GImbrailo — CHANGES_REQUESTED** [review]
There is a failing job with the website

```


20:33:12.212 | To address all issues possible (including breaking changes), run:
-- | --
20:33:12.212 | npm audit fix --force
20:33:12.213 |  
20:33:12.213 | Some issues need review, and may require choosing
20:33:12.213 | a different dependency.
20:33:12.213 |  
20:33:12.213 | Run `npm audit` for details.
20:33:12.668 | npm WARN config tmp This setting is no longer used.  npm stores temporary files in a special
20:33:12.668 | npm WARN config location in the cache, and they are managed by
20:33:12.668 | npm WARN config     [`cacache`](http://npm.im/cacache).
20:33:12.679 |  
20:33:12.679 | > taqueria_website@0.6.2 build
20:33:12.679 | > docusaurus parse && docusaurus glossary && docusaurus build && cp redirects.txt build/_redirects
20:33:12.679 |  
20:33:15.191 |  
20:33:15.191 | ! No option for terms directory found, using default directory "./docs/terms/"
20:33:15.191 |  
20:33:15.197 | Replacing patterns with <Term />
20:33:15.213 | Iterate through the .md(x) files, looking for term patterns
20:33:15.245 | ✓ 0 term replacements completed.
20:33:17.677 |  
20:33:17.678 | ! No option for terms directory found, using default directory "./docs/terms/"
20:33:17.678 |  
20:33:17.682 | Alphabetical list of terms
20:33:17.697 | ! No glossaryTermPatterns were specified to filter terms by type.
20:33:17.698 | ! Glossary file does not exist in path: "/opt/buildhome/repo/website/docs/glossary.md". Creating...
20:33:17.699 | » Parsing terms in the glossary
20:33:17.702 | Iterate through the .md(x) files, looking for term patterns
20:33:17.707 | ! A single file to be parsed is given in option "docsDir": "/opt/buildhome/repo/website/docs/glossary.md"
20:33:17.707 | ✓ 0 term replacements completed.
20:33:17.707 | » Glossary is updated.
20:33:17.708 | ✓ 1 terms found.
20:33:19.829 | [INFO] [en] Creating an optimized production build...
20:33:20.114 |  
20:33:20.115 | ! No option for terms directory found, using default directory "./docs/terms/"
20:33:20.115 |  
20:33:20.958 | ℹ Compiling Client
20:33:20.985 | ℹ Compiling Server
20:33:43.330 | ✔ Client: Compiled with some errors in 22.37s
20:33:43.336 |  
20:33:43.336 |  
20:33:43.336 | SyntaxError: /opt/buildhome/repo/website/docs/release-notes.mdx: Unexpected token, expected "{" (1:7)
20:33:43.336 | > 1 \| export TAQ_LIGO_IMAGE="ligolang/ligo:0.60"
20:33:43.337 | \|        ^
20:33:43.337 | 2 \| taq compile hello-tacos.mligo
20:33:43.337 | 3 \| ```
20:33:43.337 | at parser.next (<anonymous>)
20:33:43.337 | at normalizeFile.next (<anonymous>)
20:33:43.337 | at run.next (<anonymous>)
20:33:43.337 | at transform.next (<anonymous>)
20:33:43.337 | at new Promise (<anonymous>)
20:33:43.337 | [ERROR] Client bundle compiled with errors therefore further build is impossible.
20:33:43.394 | Failed: build command exited with code: 1
20:33:44.282 | Failed: an internal error occurred


```

File: website/docs/release-notes.mdx [hunk 1]
```diff
@@ -2,6 +2,274 @@
...
+Many plugins use docker images to render functionality. For instance, the v0.28.0 ligo plugin uses the `ligolang/ligo:0.57.0` image under the hood.
+
+Configure Taqueria alternatively by using the following environment variables:
+
+- TAQ_FLEXTESA_IMAGE
+- TAQ_LIGO_IMAGE
+- TAQ_ARCHETYPE_IMAGE
+
+For instance, to use LIGO v0.60 when compiling, do the following:
+```
```
Comment (GImbrailo) [inline website/docs/release-notes.mdx:74]: We need a line break between line 73 and 74. Fixing that got the build passing for me locally

**Review by GImbrailo — CHANGES_REQUESTED** [review]

**Comment by egarson** [comment]
recheck

**Review by GImbrailo — APPROVED** [review]
LGTM

**Pull request merged**